cmd/kots/cli/admin-console-generate-manifests.go

```diff
@@ -39,8 +50,20 @@ func AdminGenerateManifestsCmd() *cobra.Command {
 				HTTPSProxyEnvValue:   v.GetString("https-proxy"),
 				NoProxyEnvValue:      v.GetString("no-proxy"),
 				IncludeMinio:         v.GetBool("with-minio"),
+				MigrateToMinioXl:     migrateToMinioXl,
+				CurrentMinioImage:    currentMinioImage,
 				IsMinimalRBAC:        v.GetBool("minimal-rbac"),
 				AdditionalNamespaces: v.GetStringSlice("additional-namespaces"),
+				IsOpenShift:          isOpenShift,
+				IsGKEAutopilot:       isGKEAutopilot,
+
+				RegistryConfig: kotsadmtypes.RegistryConfig{
```

Comment:
you need to use the `getRegistryConfig` function which has specific logic for reading the registry flags. (check the other commands for reference)

Reply:
appears that function requires a kubernetes context.  thinking we can set defaults using the flag, and if we get a context, then use that function to update.